src/screens/error/ErrorScreen.tsx

```diff
@@ -14,7 +14,7 @@ const log = Minilog('ErrorScreen')
 /**
  * Typings
  */
-interface ErrorScreenProps {
+export interface ErrorScreenProps {
```

Comment:
Seems like you exported this interface but did not use it elsewhere. Is this intended?

Reply:
You're right I did use it at first. The export is not needed anymore